test/change_streams/unified/change-streams-clusterTime.json

```diff
@@ -28,7 +28,6 @@
       "minServerVersion": "4.0.0",
       "topologies": [
         "replicaset",
-        "sharded-replicaset",
```

Comment:
Should we remove "sharded-replicaset" from a .py file somewhere?

Reply:
Done